### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by debs-obrien** [teambit/bit#4449] [General]
Welcome everyone. I am taking some of the questions people ask on our community slack channel and posting the answers here to make it easier to find. 

Feel free to [join our Discord channel](https://discord.bit.cloud/) or use the discussions to help you find the answer to your question. Have fun with Bit and remember we are here if you need anything so just reach out.

Don't forget you can also:
- check out or [docs](https://bit.dev/docs/intro) for more details
- go through our [tutorial](https://bit.dev/docs/quick-start) to get started with Bit
- see our [resources](https://harmony-docs.bit.dev/resources/videos) for a list of videos, podcasts, live streams and more.
- see a [collection of demo apps](https://bit.dev/learn-bit-react/~collections) made with bit components

**Comment by Marvis07** [comment]
Hi

**Comment by hollyos** [comment]
Hey all! The Slack links here and on the site no longer work.

**Reply by omerpri** [reply]
Thanks for letting up know, the updated link is:
https://join.slack.com/t/bit-dev-community/shared_invite/zt-1el4b4sb5-FpgRefqUiZbUH9lvnHgwRg

**Reply by JoshK2** [reply]
Thanks, @omerpri I updated the link in the discussion.

**Reply by pcperera** [reply]
> /bit-dev-community/shared_invite/zt-1el4b4sb5-FpgRefqUiZbUH9lvnHgwRg

This slack link no longer seems to work.

**Reply by omerpri** [reply]
Thank Priyanka for flagging this, the updated link: https://join.slack.com/t/bit-dev-community/shared_invite/zt-1ccd6k1nn-wIKOlqNNUEypmmO9bUujUg

**Reply by ggros-septeo** [reply]
Hi guys,

Would it be possible to get an updated link for Slack.

Thanks :)

**Reply by JoshK2** [reply]
Hi @ggros-septeo ,
Here is the new link: https://join.slack.com/t/bit-dev-community/shared_invite/zt-1vq1vcxxu-CEVobR1p9BurmW8QnQFh1w

**Reply by YanZiSen** [reply]
hi, This slack link no longer seems to work.Can you give a new link? @JoshK2 @debs-obrien .

**Reply by StarHosea** [reply]
@debs-obrien Hi , Can you give a new link ?

**Reply by utillity** [reply]
Hi! None of the links to the slack community are working!

**Reply by JoshK2** [reply]
> Hi! None of the links to the slack community are working!

Hey, you can join our community Slack using this link: https://slack.bit.cloud/

**Reply by davidfirst** [reply]
I edited the original message. We moved from Slack to Discord.
